src/main/java/uk/gov/hmcts/reform/bulkscanning/config/AccessTokenConfiguration.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.bulkscanning.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.List;
+
+@ConfigurationProperties("accesstoken")
+public class AccessTokenConfiguration {
+    private List<TokenConfig> serviceConfig;
+
+    public List<TokenConfig> getServiceConfig() {
+        return serviceConfig;
+    }
+
+    public void setServiceConfig(List<TokenConfig> serviceConfig) {
+        this.serviceConfig = serviceConfig;
+    }
+
+    public static class TokenConfig {
```

Comment:
To be discussed tomorrow, but this PR would be simpler if we used a single container for all our services; we wouldn't need this configuration, or a parameterized endpoint or exceptions in case they ask for a container that doesn't exist.

Reply:
Yes I agree it would be simpler.Current implementation is as per discussion with Salman.I will have a chat with him to see if we can make it as a single container.